packages/block-editor/src/components/publish-date-time-picker/README.md

```diff
@@ -0,0 +1,54 @@
+# `PublishDateTimePicker`
+
+`<PublishDateTimePicker />` is a component used to select the date and time that
+a post will be published. It wraps the `<DateTimePicker />` component found in
+`@wordpress/components` and adds additional post-specific controls.
+
+See [the documentation for
+DateTimePicker](https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/date-time)
```

Comment:
Tiny nit: I think we can use an absolute path to the readme file to get links in the handbook to be directed to other pages within the handbook, rather than redirecting to the Gutenberg repo? E.g. links like the following: https://github.com/WordPress/gutenberg/blob/b964485122fa05ca305d544946ade1d3c1eeabc4/packages/components/CONTRIBUTING.md#L428

Ideally, for someone browsing in the handbook they'd ultimately be directed to: https://developer.wordpress.org/block-editor/reference-guides/components/date-time/

Reply:
Cool! I'll do that.